features/step_definitions/planning_application_steps.rb

```diff
@@ -105,7 +105,7 @@
     And I choose "Yes" for "Do you agree with the recommendation?"
     And I press "Save"
     And I press "Publish determination"
-    And I press "Determine application"
+    And I press "Publish determination"
```

Comment:
Guessing one is a link and then one is the form action?

Reply:
Yes.

The first "Publish determination" is on the "Planning Application" page...
![Screenshot 2022-10-25 at 14 43 11](https://user-images.githubusercontent.com/1710795/197789857-7937f1df-32fc-47b0-a195-a23f904314a5.png)

The second "Publish determination" is the button on the "review and publish determination" page made in this PR..